front/style/pages/_adherent-dashboard.scss

```diff
@@ -0,0 +1,97 @@
+.dashboard {
```

Comment:
can you prefix with `adherent-` pls? `.dashboard` is too generic...

Reply:
renamed